fuse-products/src/main/java/software/tnb/product/csb/application/LocalSpringBootApp.java

```diff
@@ -89,11 +89,15 @@ public void stop() {
         if (logStream != null) {
             logStream.stop();
         }
+        if (log != null) {
+            log.save();
+        }
         if (appProcess != null) {
             LOG.info("Stopping integration {}", name);
             if (appProcess.isAlive()) {
                 LOG.debug("Killing integration process");
                 appProcess.destroy();
+                WaitUtils.waitFor(() -> !isReady(), 50, 100, "Waiting until the process is stopped");
```

Comment:
do you think 5 seconds is enough?

Reply:
it seems the apps were stopped each time in about ~50ms, but thinking about it now, there is a default timeout in camel: https://camel.apache.org/manual/graceful-shutdown.html , so we should probably have a bigger timeout than that, so I could increase it to 60 seconds, wdyt?

Reply:
yes it happens that the app need a little bit more, but I think 60 seconds is enough, thanks